sequencer/batchbuilder.go

```diff
@@ -243,23 +243,61 @@ func (s *Sequencer) closeSequence(ctx context.Context) error {
 // in case it's enough blocks since last GER update, long time since last batch and sequence is profitable
 func (s *Sequencer) shouldCloseSequenceInProgress(ctx context.Context) bool {
 	// Check if GER needs to be updated
-	numberOfBlocks, err := s.state.GetNumberOfBlocksSinceLastGERUpdate(ctx, nil)
-	if err != nil && err != state.ErrNotFound {
-		log.Errorf("failed to get last time GER updated, err: %v", err)
+	blockNum, mainnetExitRoot, err := s.state.GetBlockNumAndMainnetExitRootByGER(ctx, s.sequenceInProgress.GlobalExitRoot, nil)
```

Comment:
it should trigger `object not found` in the init stage.
`GlobalExitRoot = 0x0000...` there is no the related main_exit_root.

Reply:
`2022-08-11T09:27:08Z    ERROR   sequencer/batchbuilder.go:248   failed to get mainnetExitRoot and blockNum by ger, err: object not found%!(EXTRA string=
/src/log/log.go:106 github.com/0xPolygonHermez/zkevm-node/log.appendStackTraceMaybeArgs()
/src/log/log.go:164 github.com/0xPolygonHermez/zkevm-node/log.Errorf()
/src/sequencer/batchbuilder.go:248 github.com/0xPolygonHermez/zkevm-node/sequencer.(*Sequencer).shouldCloseSequenceInProgress()
/src/sequencer/batchbuilder.go:29 github.com/0xPolygonHermez/zkevm-node/sequencer.(*Sequencer).tryToProcessTx()
/src/sequencer/sequencer.go:132 github.com/0xPolygonHermez/zkevm-node/sequencer.(*Sequencer).Start.func1()
)`